### PR TITLE
[EASI-4141] Update POCs for MDM

### PR DESCRIPTION
**Pull request by OddTomBrooks**
# EASI-4141

## Changes and Description

- Created update migration
  - Removed old POCs
  - Added new POcs

## How to test this change

- Bring up fresh backend containers
- Inspect the possible_operational_solution_contact table
- Assert that entries for `Felicia Addai` and `Miyani `Treva` are removed
- Assert that entries for `Sarah Okorafor` and `Luiz Lee` are added

## PR Author Review Checklist

- [ ] Met the ticket's acceptance criteria, or will meet them in a subsequent PR.
- [ ] Added or updated tests for backend resolvers or other functions as needed.
- [ ] Added or updated client tests for new components, parent components, functions, or e2e tests as necessary.
- [ ] Updated the [Postman Collection](../MINT.postman_collection.json) if necessary.

## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.

### CODE REVIEW

File: migrations/V152__Update_Possible_Operational_Solution_Contacts.sql [hunk 1]
```diff
@@ -0,0 +1,32 @@
+-- Remove Hung Van and Yolanda Villanova as Solution Contacts
```
Comment (StevenWadeOddball) [inline migrations/V152__Update_Possible_Operational_Solution_Contacts.sql:1]: Can you update this please? This is the comment from the source migration.

File: migrations/V152__Update_Possible_Operational_Solution_Contacts.sql [hunk 2]
```diff
@@ -0,0 +1,32 @@
+-- Remove Hung Van and Yolanda Villanova as Solution Contacts
+DELETE FROM possible_operational_solution_contact
+WHERE NAME IN ('Felicia Addai', 'Miyani Treva');
```
Comment (StevenWadeOddball) [inline migrations/V152__Update_Possible_Operational_Solution_Contacts.sql:3]: Do you know if these contacts were a POC for any other solution, or just this one? If just this one, this looks good.

File: migrations/V152__Update_Possible_Operational_Solution_Contacts.sql (hunk 1 above)
Comment (OddTomBrooks) [reply migrations/V152__Update_Possible_Operational_Solution_Contacts.sql]: Good catch. Done. 👍

File: migrations/V152__Update_Possible_Operational_Solution_Contacts.sql [hunk 3]
```diff
@@ -0,0 +1,34 @@
+-- Remove old entries from Solution Contacts
+DELETE FROM possible_operational_solution_contact
+WHERE NAME IN ('Felicia Addai', 'Miyani `Treva`');
```
Comment (StevenWadeOddball) [inline migrations/V152__Update_Possible_Operational_Solution_Contacts.sql:3]:
```suggest
WHERE NAME IN ('Felicia Addai', 'Miyani Treva');
```

The current SQL doesn't remove `Miyani Treva`

**Review by StevenWadeOddball — CHANGES_REQUESTED** [review]

File: migrations/V152__Update_Possible_Operational_Solution_Contacts.sql (hunk 3 above)
Comment (OddTomBrooks) [reply migrations/V152__Update_Possible_Operational_Solution_Contacts.sql]: I must have accidentally hit the backtick key on their name when copying it for PR writeup or when searching through other migrations. Fixed.

**Review by StevenWadeOddball — APPROVED** [review]

**Pull request merged**